libraries/go/webhook.go

```diff
@@ -52,6 +80,17 @@ func NewWebhookRaw(secret []byte) (*Webhook, error) {
 	}, nil
 }
 
+func NewWebhookWithOptions(method WebhookMethod, options *WebhookOptions) (*Webhook, error) {
+	if options.KeyObject.Key == nil {
```

Comment:
Maybe pass WebhookOptions by value so you don't need to do yet another nil check?